android/fakedapp/src/main/java/com/solana/mobilewalletadapter/fakedapp/MainViewModel.kt

```diff
@@ -91,6 +91,10 @@ class MainViewModel(application: Application) : AndroidViewModel(application) {
         }
 
         val signedTransactions = localAssociateAndExecute(sender) { client ->
+            val authorized = doReauthorize(client)
```

Comment:
So if I'm grokking this right, the clients will need to essentially make an authorize call before each operation? The assumption being that in many cases, the auth state will still be valid and thus will come back as authorized=true very quickly?

Reply:
And on a related note, our convenience wrappers are going to largely abstract this away, no?

Reply:
+1 that looks good to me if we do a `reauthorize()` before each operation, it's easy to abstract away in the convenience lib as well.

I'm guessing we wouldn't need to expose `reauthorize()` at all in the convenience libs, just `authorize()`?

Reply:
Yep, after a session is established, it'll need either an `authorize` or a `reauthorize` call before any of the privileged method calls. I expect the convenience wrappers can abstract this away from consumers

Reply:
Yeah this sounds great. I'll wait to what you do with my precious ktx lib before approving :)